app/actionUpdate/get.controller.js

```diff
@@ -25,11 +60,14 @@ const getActionUpdate = async (
       )
     )
     const backUrl = `${removeUrlLevels(path, 2)}`
-    const { intervention, description } = action
-    const actionText = intervention ? await getInterventionText(tokens, intervention) : description
-    const { motivationList } = await getMotivation(action, body, tokens).catch(error =>
-      catchAndReThrowError(`Could not retrieve motivation list`, error)
-    )
+    const { intervention, description, progress, created = '' } = action
+    const sausage = await Promise.all([
```

Comment:
Are we really accepting your sausage?

Reply:
🤦‍♂Coding while clearly unwell has its drawbacks...  
Removing now 👍